ui/app/templates/vault/cluster/access/mfa/configure.hbs

```diff
@@ -0,0 +1,34 @@
+<PageHeader as |p|>
+  <p.levelLeft>
+    <h1 class="title is-3">
+      Multi-factor authentication
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+
+<div class="box is-fullwidth is-sideless is-flex">
+  <p>
+    Configure and enforce multi-factor authentication (MFA) for users logging into Vault, for any
+    <br />
+    authentication method.
+    <LearnLink @path="/tutorials/vault/usage-metrics">
+      Learn more
+    </LearnLink>
+  </p>
+  <button type="submit" class="button is-primary is-flex-end">
+    Configure MFA
+  </button>
+</div>
+<div class="box is-fullwidth is-shadowless">
+  <p>
+    <b>Step 1:</b>
+    Set up an MFA configuration using one of the methods; TOTP, Okta, Duo or Pingid.
+  </p>
+  <p>
+    <b>Step 2:</b>
+    Set up an enforcement to map the MFA configuration to your chosen auth method(s).
+  </p>
+</div>
+<div class="box is-fullwidth is-shadowless">
```

Comment:
What is this div for if it's not being used? e.g. there is nothing inside of it.

Reply:
Was waiting to get the image, update the div.